pkg/scheduler/framework/preemption/preemption.go

```diff
@@ -378,6 +378,7 @@ func (ev *Evaluator) prepareCandidate(ctx context.Context, c Candidate, pod *v1.
 				return
 			}
 		}
+		klog.V(2).InfoS("Preemptor pod preempts the victim pod", "preemptor", klog.KObj(pod), "victim", klog.KObj(victim), "node", c.Name())
```

Comment:
@Huang-Wei do you think we should have a different message for preemptions of waiting pods?

Also, use past tense: `Preemptor Pod preempted victim Pod`

Reply:
> We should. A waitingPod is actually not physically deleted, but unreserved from internal cache. So let's add the message below `waitingPod.Reject()`:
> 
> ```
> klog.V(2).InfoS("Preemptor pod rejected a waiting pod", "preemptor", klog.KObj(pod), "waitingPod", klog.KObj(victim), "node", c.Name())
> ```
> 
> @HirazawaUi please also move the message to the end of `else` section; otherwise it's printed for waitingPod case as well.

Thanks for remind, has made corrections